bLactam_MIC_Rscripts/AAtable_To_MIC_MM_RF_EN_2.R

```diff
@@ -2,13 +2,7 @@
 
 ## Function PBP_AA_TO_MIC2
 
-PBP_AA_TO_MIC2<- function(cwd){
-  dbdir="/scicomp/groups/OID/NCIRD/DBD/RDB/Strep_Lab/External/share/PBP_AA_to_MIC/newDB/"
-  libpath="/scicomp/groups/OID/NCIRD/DBD/RDB/Strep_Lab/External/share/PBP_AA_to_MIC/Rlib"
-  #x1=.libPaths()
-  x2=c(libpath, "/usr/lib64/R/library", "/usr/share/R/library")
-  .libPaths(x2)
-  #.libPaths(c(.libPaths(), libpath))
+PBP_AA_TO_MIC2<- function(cwd, dbdir){
```

Comment:
passing the install.   I moved BLOSUM62.csv to newdb directory.  Not sure if it was the right thing to do here, but I essentially wanted to pass the directory where this file leaves as parameter to the R function